docs/osf-trivia-list/dasharo-pro-package.md

```diff
@@ -72,3 +72,19 @@ be flashed with proprietary firmware using this method.
 
 The keys/credentials have no other usages outside Dasharo Tools Suite
 and have no impact on other components of the system.
+
+## How can I unsubscribe from a DPP newsletter
```

Comment:
@PLangowski did you see https://github.com/knadh/listmonk/issues/1407
Commits 2 weeks ago

Reply:
I see. We will probably have to wait for the new listmonk release to have that

Reply:
Now I see that this was added to listmonk a long time ago. The one from 2 weeks before is some fork. It looks like this is a different issue

Reply:
I do not agree, the description states the same what you added in the section below. If you can document what's the problem and add the issue in the Listmonk GH, that would be publicly visible, then we can reference it here.

Reply:
https://github.com/Dasharo/docs/compare/6c1bda47f772c59be51823ac7711b4d95407872b..0018aa50be6d1d845ad7baa725f3ce9e688b8af8